tests/main/install-hook-can-run-svcs/test-snapd-install-hook-runs-svc/bin/svc.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+echo "snap-svc-did-this" > "$SNAP_DATA/data"
+
+sleep 1000
```

Comment:
(nitpick^3) you could use `sleep infinity` here (it's a thing :)

Reply:
I try to avoid using sleep infinity when I know that something will be manually stopped or should be manually stopped by something else just in case there are coding errors and something ends up blocking on the sleep, I would rather have it just take a long time than reach a deadlock condition, but if you prefer I can certainly bundle in the infinity change with the other comments

Reply:
That's fine then, thanks for the context! I assumed 1000 means "a-very-long-time". It's fine to keep.